scripts/override_dates_in_dist.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -euo pipefail
+
+DIST_FILE_NAME=$(basename $1)
+DIST_FILE_ABSPATH=$(readlink -f $1)
+
+#
+# The timestamp of the last commit. We need to remove the last part of the
+# timestamp because the 'touch' command doesn't work with it.
+#
+LAST_COMMIT_DATETIME=$(git log -1 --format=%cI | cut -d'+' -f1)
```

Comment:
I will need to convert this time to UTC instead of stripping the `+XX:XX` or `-XX:XX` from the end.

Reply:
This is definitely starting to sound like writing it in Java may be easier.  We can write libraries in a `buildSrc` dir and gradle will compile it and make it available to the build process which makes life simpler.  I'd have to look up the exact details again but it's pretty straight forward, just a matter of getting the right directory structure.